service/libp2p_pubsub.go

```diff
@@ -100,8 +101,9 @@ func (ps *Libp2pPubsub) Subscribe(bootstrap bool, h MessageHandler) error {
 		if err := proto.Unmarshal(msg, wb); err != nil {
 			return nil, fmt.Errorf("unmarshaling message: %v", err)
 		}
-		err := h.WinsHandler(wb)
-		if err != nil {
+		ctx, cls := context.WithTimeout(context.Background(), time.Second*30)
```

Comment:
We didn't have a timeout before. How does this change is seen from the client's perspective? Is it expected to see more errors?